src/main/java/Duke.java

```diff
@@ -1,10 +1,90 @@
-public class Duke {
+import java.util.Scanner;
+import java.util.ArrayList;
+
+public class
+Duke {
     public static void main(String[] args) {
```

Comment:
Your main method is far too long, the recommendation according to the [textbook](https://nus-cs2113-ay2223s2.github.io/website/se-book-adapted/chapters/codeQuality.html#avoid-long-methods) is 30 LOC. Please try to modularize your main method.